test/JDBC/expected/sys-system_sql_modules-dep-vu-prepare.out

```diff
@@ -17,5 +17,5 @@ END
 GO
 
 CREATE VIEW sys_system_sql_modules_dep_vu_prepare_v1 AS
-    SELECT uses_ansi_nulls FROM sys.system_sql_modules WHERE object_id = OBJECT_ID('sys.fn_listextendedproperty')
+    SELECT uses_ansi_nulls, definition FROM sys.system_sql_modules WHERE object_id = OBJECT_ID('sys.fn_listextendedproperty')
```

Comment:
include object whosse body is of length more than 4000

Reply:
We don't support object definition of system objects. That would be NULL.